backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -771,7 +629,9 @@ private void deleteRepositories(Session session) {
         } catch (Exception ex) {
           LOGGER.warn("DeleteEntitiesCron : deleteRepositories : Exception: ", ex);
         } finally {
-          transaction.commit();
+          if (transaction != null) {
+            transaction.commit();
```

Comment:
We should not commit something that had an exception during execution.

Reply:
FIxed